cypress/cypress/integration/record.spec.ts

```diff
@@ -578,6 +588,141 @@ describe("Record", () => {
       cy.get("[data-cy=done-btn]").should("exist");
       cy.get("[data-cy=done-btn]").trigger("mouseover").click();
     });
+
+    it("End-of-category page should display unordered list of follow up questions", () => {
```

Comment:
this is super minor, but good to get used to the grammar of BDD testing

DESCRIBE:  recording page
IT: displays a list of followup questions at end of category

...to me is the english phrasing that would make sense